dev/io.openliberty.jakarta.pages.tld.3.0/bnd.bnd

```diff
@@ -0,0 +1,42 @@
+#*******************************************************************************
+# Copyright (c) 2017, 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.2
+-nouses=true
+
+Bundle-SymbolicName: io.openliberty.jakarta.pages.tld.3.0.javaee
+
+-pom: artifactid=io.openliberty.jakarta.pages.tld.3.0
+
+jakartaeeMe: true
+jakartaFinalJarName: io.openliberty.jakarta.pages.tld.3.0.jar
+jakartaFinalBundleId: io.openliberty.jakarta.pages.tld.3.0
+
+# Only publish the Jakarta one
+publish.wlp.jar.include: io.openliberty.jakarta.pages.tld.3.0.jar
+
+Export-Package: \
+    org.apache.taglibs.standard.tag.common.core;version="1.0.0", \
+    org.apache.taglibs.standard.tag.common.xml;version="1.0.0", \
+    org.apache.taglibs.standard.tag.el.core;version="1.0.0", \
+    org.apache.taglibs.standard.tag.el.fmt;version="1.0.0", \
+    org.apache.taglibs.standard.tag.el.sql;version="1.0.0", \
+    org.apache.taglibs.standard.tag.el.xml;version="1.0.0", \
+    org.apache.taglibs.standard.tag.rt.core;version="1.0.0", \
+    org.apache.taglibs.standard.tag.rt.fmt;version="1.0.0", \
+    org.apache.taglibs.standard.tag.rt.sql;version="1.0.0", \
+    org.apache.taglibs.standard.tag.rt.xml;version="1.0.0"
```

Comment:
Spoke with my team -- Should this be 1.0.0 because it's a new bundle? Or should it be 2.0 since these are the EE9 versions of the classes (albeit transformed)

Reply:
Ah, the previous bundle used 1.0.0

Reply:
Don't confuse bundle version and package versions. The bundle version specified here is `1.2`. This comment appears to be referring to the exported package versions, which are currently `1.0.0`. I don't know what version either one should be but don't confuse the two.

Reply:
Why is the bundle version 1.2?